src/inspector_agent.cc

```diff
@@ -215,7 +215,7 @@ class JsBindingsSessionDelegate : public InspectorSessionDelegate {
     Local<Value> argument = v8string.ToLocalChecked().As<Value>();
     Local<Function> callback = callback_.Get(isolate);
     Local<Object> receiver = receiver_.Get(isolate);
-    callback->Call(env_->context(), receiver, 1, &argument);
+    static_cast<void>(callback->Call(env_->context(), receiver, 1, &argument));
```

Comment:
Agreed if we document this somewhere as an idiom `static_cast<void> === #pragma GCC unused-result ignored`
Because a quick search only found one other use, in this file in line 210@master 352@here.